### DISCUSSION TITLE
Self update on Windows

### DISCUSSION THREAD

**Discussion by Drapersniper** [topgrade-rs/topgrade#385] [Q&A] [answered]
Is there a way to either:
 - Stop the GitHub downloaded binaries from deleting itself on Windows?
 - If not, Is there a way to currently disable the self-update on Windows? 
 
 Issue: 
 - Every time topgrade is run (Admin and non-admin terminal) the binary deletes itself (maybe it is moved but never moved back).

**Comment by s34m** [comment]
This shouldn't happen. Could you please open an issue for that and maybe post the topgrade output

**Comment by Drapersniper** [comment, marked as answer]
So turns out i had the rename option enabled in the config file - disabling that the file is no longer nuked
